doc/packet_trimming/packet-trimming-design.md

```diff
@@ -351,8 +358,22 @@ In order to use trimming configuration in ACL rule, a dedicated ACL table type m
 
 Existing infrastructure will be reused.
 
-Flex counter groups `port/queue` will be extended with a new SAI attributes  
-`SAI_PORT_STAT_TRIM_PACKETS/SAI_QUEUE_STAT_TRIM_PACKETS` respectively.
+Flex counter groups `port/queue` will be extended with a new SAI attributes:
+* `SAI_QUEUE_STAT_TRIM_PACKETS`
```

Comment:
will all queues supporting these counters including the service ports?

Reply:
@r12f all ports will have a support of a drop counters